docs/reference/appointment-types.md

```diff
@@ -0,0 +1,1192 @@
+---
+{
+  "pageClass": "reference-page"
+}
+---
+
+# Appointment types
+
+:::::panel
+
+:::: left
+
+You can use the `/v2/vytemes/:nickname/types` endpoints to manage the availabilities of your users.
```

Comment:
the appointment types of your users

Reply:
done a004104d92e7775d157df1b7c2050653f7e7f587